Makefile

```diff
@@ -103,6 +104,7 @@ $(playbook_TARGETS): %: %.yml
 
 -include Makefile_services.mk
 -include Makefile_reboot.mk
+-include Makefile_deploy.mk
```

Comment:
It seems the Makefile name is different. It's missing the ".mk" suffix

Reply:
fixed